src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2449,6 +2464,16 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
                 }
             }
 
+            switch ((conv1.Kind, conv2.Kind))
```

Comment:
The proposed spec changes covers "better function member" and is implemented by `BetterFunctionMember` above. But I don't a spec change for better conversion from expression. I think this change is sensible, but let's spec it too.

Reply:
I haven't found a case where this change is still needed so I've removed it.